sonar-project.properties

```diff
@@ -1,14 +1,14 @@
 sonar.projectKey=sap_cloud-commerce-spartacus-storefront
 sonar.projectName=Cloud commerce spartacus storefront
 
-sonar.sources=projects/assets,projects/cds,projects/core,projects/storefrontstyles,projects/storefrontlib,projects/schematics,projects/incubator,projects/vendor
+sonar.sources=projects/assets,projects/cds,projects/core,projects/storefrontstyles,projects/storefrontlib,projects/schematics,projects/incubator,projects/vendor,feature-libs/my-account
 sonar.exclusions=**/node_modules/**
 
-sonar.tests=projects/cds,projects/core,projects/storefrontlib
+sonar.tests=projects/cds,projects/core,projects/storefrontlib,feature-libs/my-account
 sonar.test.inclusions=**/*.spec.ts
 
-sonar.typescript.lcov.reportPaths=coverage/cds/lcov.info,coverage/core/lcov.info,coverage/storefront/lcov.info
-sonar.javascript.lcov.reportPaths=coverage/cds/lcov.info,coverage/core/lcov.info,coverage/storefront/lcov.info
+sonar.typescript.lcov.reportPaths=coverage/cds/lcov.info,coverage/core/lcov.info,coverage/storefront/lcov.info,feature-libs/my-account/lcov.info
```

Comment:
I think this doesn't match the path you have in karma config

Reply:
Thanks, I set it to `coverage/my-account/lcov.info`